lighthouse-core/test/chromium-web-tests/download-devtools.sh

```diff
@@ -14,6 +14,7 @@ then
   cd "$DEVTOOLS_PATH"
 
   # Update to keep current
+  git clean -fd
   git pull -f origin master
   gclient sync
```

Comment:
```suggestion
  gclient sync --delete_unversioned_trees --reset
```


```
  -D, --delete_unversioned_trees
                        Deletes from the working copy any dependencies that
                        have been removed since the last sync, as long as
                        there are no local modifications. When used with
                        --force, such dependencies are removed even if they
                        have local modifications. When used with --reset, all
                        untracked directories are removed from the working
                        copy, excluding those which are explicitly ignored in
                        the repository.
  -R, --reset           resets any local changes before updating (git only)
```

reset handles git changes inside deps.